scss/_variables.scss

```diff
@@ -837,7 +837,14 @@ $zindex-modal:                      1050 !default;
 $zindex-popover:                    1060 !default;
 $zindex-tooltip:                    1070 !default;
 // scss-docs-end zindex-stack
-
+$z-index-levels: (
+  0,
```

Comment:
I'd be in favor of `-1, 0, 1, 2` as values since they're the most used values (see  [Web Almanac's CSS section about `z-index`](https://almanac.httparchive.org/en/2019/css#z-index) and [its results spreadsheet](https://docs.google.com/spreadsheets/d/1uFlkuSRetjBNEhGKWpkrXo4eEIsgYelxY-qR9Pd7QpM/edit#gid=0)).

Reply:
Thanks for your help